tune-threads/utilities.lisp

```diff
@@ -0,0 +1,50 @@
+;;;; tune-threads/tune-threads.lisp
+;;;;
+;;;; Author: John Lapeyre
+
+(in-package #:qvm-tune-threads)
+
+(defun mean-dev-from-sums (x-sum x-sq-sum n)
+  "Given a sum X-SUM and sum of squares X-SQ-SUM of N iid random samples, compute the mean and the standard deviation"
+  (let ((x-mean (/ x-sum n)))
+    (list x-mean (sqrt (- (/ x-sq-sum n) (expt x-mean 2))))))
+
+(defun findmin (list)
+  "Return a list whose two elements are the minimum value of LIST and the index of that value.
+If the minimum is degenerate, the fist index is returned."
+  (do* ((min-val (car list))
+        (min-ind 0)
+        (list (cdr list) (cdr list))
+        (next-val (car list) (car list))
+        (ind 0 (1+ ind)))
+       ((null list) (list min-val min-ind))
+    (when (< next-val min-val)
+      (setf min-val next-val)
+      (setf min-ind (1+ ind)))))
+
+(defun norm-min (list)
+  "Normalize LIST by its minimum element"
+  (let ((min (apply #'min list)))
+    (map 'list #'(lambda (x) (/ x min)) list)))
```

Comment:
Maybe `(mapcar ...)` here instead of `(map 'list ...)`?  Alternatively, if the idea is to allow the `list` argument to be any sequence type, rename it to reflect that.

This is a weak personal preference that others may disagree with, so if you prefer `(map 'list ...)`, feel free to ignore.

Reply:
I'll look into.

Reply:
I already assume `list` is a list when finding the minimum. Using mapcar seems reasonable.